src/Geometry.cpp

```diff
@@ -1040,9 +1064,9 @@ std::vector<int> Geometry::getFSRsToMaterialIDs() {
  *          are read from file to avoid unnecessary segmentation.  
  * @param FSR_keys_map map of FSR keys to FSR data
  */
-void Geometry::setFSRKeysMap(std::unordered_map<std::size_t, fsr_data> 
+void Geometry::setFSRKeysMap(std::unordered_map<std::size_t, fsr_data>*
```

Comment:
Can we add this to the list of SWIG ``%ignore`` routines? I know this isn't a new method, but since it came up in our review here I think it would be good to add.